containers/datalab/Dockerfile.in

```diff
@@ -106,8 +106,11 @@ ADD build/lib/GCPData-0.1.0.tar.gz /datalab/lib/
 ADD build/lib/GCPDataLab-0.1.0.tar.gz /datalab/lib/
 ADD build/metadata /datalab/metadata
 ADD build/web /datalab/web
+ADD nbconvert /datalab/nbconvert
 ADD content/ /datalab
 ADD config/ipython.py /root/.ipython/profile_default/ipython_config.py
+RUN mkdir -p /root/.jupyter
```

Comment:
Is this line needed - there is a RUN jupyter notebook --generate-config step earlier.